ADVANCED_EXAMPLES.md

```diff
@@ -217,7 +217,8 @@ First, let's assume you have this stamp:
 const User = stampit.refs({ entityName: 'user' });
 ```
 
-The following stamp should be composed *last*, otherwise it won't work.
+When combined the following stamp will make any existing stamp to return promise instead of an object instance.
```

Comment:
make any existing stamp to return promise
->
make any existing stamp return a promise